bootstrap/bootstrap.go

```diff
@@ -1133,75 +1133,81 @@ func (b *Bootstrap) CheckoutPhase(ctx context.Context) error {
 			return err
 		}
 	default:
-		if b.Config.Repository != "" {
-			err := roko.NewRetrier(
-				roko.WithMaxAttempts(3),
-				roko.WithStrategy(roko.Constant(2*time.Second)),
-			).DoWithContext(ctx, func(r *roko.Retrier) error {
-				err := b.defaultCheckoutPhase(ctx)
-				if err == nil {
-					return nil
-				}
-
-				switch {
-				case shell.IsExitError(err) && shell.GetExitCode(err) == -1:
-					b.shell.Warningf("Checkout was interrupted by a signal")
-					r.Break()
-
-				case errors.Is(err, context.Canceled):
-					b.shell.Warningf("Checkout was cancelled")
-					r.Break()
-
-				case errors.Is(ctx.Err(), context.Canceled):
-					b.shell.Warningf("Checkout was cancelled due to context cancellation")
-					r.Break()
-
-				default:
-					b.shell.Warningf("Checkout failed! %s (%s)", err, r)
-
-					// Specifically handle git errors
-					if ge, ok := err.(*gitError); ok {
-						switch ge.Type {
-						// These types can fail because of corrupted checkouts
-						case gitErrorClone:
-						case gitErrorClean:
-						case gitErrorCleanSubmodules:
-							// do nothing, this will fall through to destroy the checkout
-
-						default:
-							return err
-						}
-					}
+		if b.Config.Repository == "" {
+			b.shell.Commentf("Skipping checkout, BUILDKITE_REPO is empty")
+			break
+		}
```

Comment:
This used to be on 1197. I've inverted the condition to decrease the indentation.